tests/expected_tests.cc

```diff
@@ -0,0 +1,39 @@
+#include "gtest/gtest.h"
+#include "stout/expected.h"
+
+stout::expected<int, std::string> divide(int divident, int divider) {
```

Comment:
These names should be `dividend` and `divisor`.

https://www.hmhco.com/blog/teaching-dividend-divisor-and-quotient-in-division#:~:text=The%20number%20that%20is%20being%20divided%20(in%20this%20case%2C%2015)%20is%20called%20the%20dividend%2C%20and%20the%20number%20that%20it%20is%20being%20divided%20by%20(in%20this%20case%2C%203)%20is%20called%20the%20divisor.

Reply:
Oops, sorry for that:) Will replace that!